src/test/java/com/gooddata/dataload/processes/DataloadProcessesTest.java

```diff
@@ -6,22 +6,33 @@
 package com.gooddata.dataload.processes;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.hamcrest.CoreMatchers;
 import org.testng.annotations.Test;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
 
+import java.util.regex.Pattern;
+
 public class DataloadProcessesTest {
 
+    private DataloadProcesses processes;
+
     @Test
     public void testDeserialization() throws Exception {
-        final DataloadProcesses processes = new ObjectMapper()
+        processes = new ObjectMapper()
                 .readValue(getClass().getResourceAsStream("/dataload/processes/processes.json"), DataloadProcesses.class);
 
         assertThat(processes, is(notNullValue()));
         assertThat(processes.getItems(), hasSize(1));
     }
 
+    @Test(dependsOnMethods = "testDeserialization")
```

Comment:
to sum up the options (even the absur ones):
1. inter test dependencies
2. make constructors package private
3. do the same deserialization second time for the toString test
4. assert toString in deserialization test

Which of those is the lesser evil? I personally incline to 3., but I am strongly personally biased against test dependencies...